### PR TITLE
New topic: statechains

### PR DESCRIPTION
**Pull request by harding**

### CODE REVIEW

File: _topics/en/statechains.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
+## true" to bold entry
+optech_mentions:
+  - title: "Bitcoin Core contributor meeting discussion topic: blind statechains"
+    url: /en/newsletters/2019/06/12/#assumeutxo
+
+  - title: Discussion about implementing statechains without schnorr or eltoo
+    url: /en/newsletters/2020/04/01/#implementing-statechains-without-schnorr-or-eltoo
+
+  - title: Transcript abuot watchtowers and their usefulness for statechains
```
Comment (bitschmidty) [inline _topics/en/statechains.md:35]: s/abuot/about

File: _topics/en/statechains.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+signer (such as previous delegates Alice or Bob).  A delegated signer
+can always spend the UTXO onchain without needing permission from the
+trusted third party, arguably making statechains less trusted than
+federated [sidechains][topic sidechains].
+
+Because anyone who was ever a delegate can trigger an onchain spend,
+statechains are designed to use the [eltoo][topic eltoo] mechanism to
+ensure an onchain spend by the most recent delegate (Carol) can take
+precedence over spends by previous delegates (Alice and Bob), assuming
+the trusted third party hasn’t colluded with a previous delegate to
```
Comment (bitschmidty) [inline _topics/en/statechains.md:74]: From the text it is not clear to me the third party's base case role in the statechain. I just understand that they cooperate in delegation operations (are they required?) and can only steal funds via collusion w/ any delegate.

**Review by bitschmidty — COMMENTED** [review]
🙏 @harding

**Comment by jnewbery** [comment]
LGTM. No additional review comments.

File: _topics/en/statechains.md (hunk 2 above)
Comment (harding) [reply _topics/en/statechains.md:74]: Rearranged and added some text to hopefully make this clearer.  Let me know if it's not.

**Comment by jnewbery** [comment]
ACK. @bitschmidty - if you're happy with https://github.com/bitcoinops/bitcoinops.github.io/pull/562#discussion_r633698222, please go ahead and merge.

**Comment by bitschmidty** [comment]
Post merge ACK, apologies for the delay @harding

**Pull request merged**